pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "mentorship-backend"
+version = "1.0.0"
+description = " Backend REST API for the Mentorship System."
+authors = ["Isabel Costa <isabelcmdcosta@gmail.com>"]
```

Comment:
can you please remove my personal email from here? I don't know which should come, but not my personal email for sure.
Perhaps you can put opensource@anitab.org for now not sure. This is for authors and maintainers fields.

Reply:
Sure! Sorry about this.